docs/iis-instrumentation.md

```diff
@@ -16,6 +16,10 @@ Install-OpenTelemetryCore
 Register-OpenTelemetryForIIS
 ```
 
+If you are using versions 0.7.0 - 1.1.0 of the OpenTelemetry .NET Automatic Instrumentation
```

Comment:
Should we provide a link to the issue or describe the behavior if this guideline is not followed?

Reply:
Updated in 02fa41ebd33cb6cc5033bcc2242f0ce817bc7a90, what do you think?